_acs-aem-commons/features/packagers/asset-packager/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: acs-aem-commons_feature
+title: Asset Packager
+description: Easily package pages and the associated assets
+redirect_from: /acs-aem-commons/features/asset-packager.html
+date: 2019-05-20
+feature-tags: administration
+initial-release: 4.2.0
+---
+
+# Purpose
+
+The Asset Packager lets you create a content package that is given a specific page path and will iterate over the content of that package to add all referenced assets into the package as well, giving you exactly what you need for a single page with all images.
+
+Helps a developer looking to:
```

Comment:
What about the use case of migrating content up from UAT to Production?  Seems like a valid and very common use case, and a little less developer-centric.

Reply:
Updated in the latest commits